index.js

```diff
@@ -49,6 +63,8 @@ function copy(src) {
 			dst = { message: src.message };
 		} else if (isBoolean(src) || isNumber(src) || isString(src)) {
 			dst = Object(src);
+		} else if (isBuffer(src)) {
+			dst = Buffer.from(src);
```

Comment:
```suggestion
		} else if (isBuffer(src) && typeof Buffer.from === 'function') {
			dst = Buffer.from(src);
```
i'm a bit concerned that merely mentioning `Buffer` would cause bundlers to bring in the `buffer` polyfill unnecessarily, which is why the `isBuffer` code is authored the way it is.

Reply:
I see, yeah not sure

Reply:
Any updates?

Reply:
I'm still concerned here - I don't think we can use `Buffer.from` without incurring a bundle size impact. I think we could use the `safe-buffer` package, but then we need to add a dependency.